tests/console/wicked_before_test.pm

```diff
@@ -27,7 +27,8 @@ sub run {
     assert_script_run('sed -e "s/^WICKED_DEBUG=.*/WICKED_DEBUG=\"all\"/g" -i /etc/sysconfig/network/config');
     assert_script_run('sed -e "s/^WICKED_LOG_LEVEL=.*/WICKED_LOG_LEVEL=\"debug\"/g" -i /etc/sysconfig/network/config');
     assert_script_run('cat /etc/sysconfig/network/config');
-    assert_script_run('clean_system=$(snapper create -p -d "clean system")');
+    my $snapshot_number = script_output('snapper create -p -d "clean system"');
```

Comment:
Be aware that serial output may be polluted with some other stuff which was sent to serial device. So I would do some parsing here.

Reply:
@rwx788 but this `script_ouput` issues should be resolved with the following PRs, no?
https://github.com/os-autoinst/os-autoinst/pull/878
https://github.com/os-autoinst/os-autoinst/pull/886

Reply:
@ldevulder I'm talking about another issue which won't be resolved by mentioned PRs as we still read output from serial device where we redirect all messages from SUT. So, for example, if you run script which writes to the serial device, this will appear in the result of script_output, even though nothing will be printed on tty. Check this ticket for better understanding: https://progress.opensuse.org/issues/25716

Reply:
@rwx788 Ok, thanks for the information.